index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Microverse Trials Survey</title>
+    <link rel="stylesheet" href="/style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+  </head>
+
+  <body>
+    <div id="container">
```

Comment:
It would be better to explain your comments politely. For example, you can use this sentence at the beginning of your comment "I think it would be better to use semantic tags".